miso-web/src/main/webapp/styles/style-full.css

```diff
@@ -65,7 +65,11 @@ pre.wrap {
 }
 
 a {
-    color: #0066cc;
+    color: #404087;
+}
+
+a:hover { 
+    text-decoration: none;

```

Comment:
Disappearing underline on hover seems strange to me. Maybe reverse this - `a { text-decoration: none} a:hover { text-decoration: underline }`?

Reply:
Fixed.